src/Command.hpp

```diff
@@ -17,22 +18,33 @@ const unsigned long DEFAULT_COMMAND_TIMEOUT = 30;
 class Command {
  public:
   Command(const std::string& command)
-      : m_cmd(command), m_timeout(DEFAULT_COMMAND_TIMEOUT) {}
+      : m_cmd(command),
+        m_timeout(DEFAULT_COMMAND_TIMEOUT),
+        m_frequence(DEFAULT_COMMAND_FREQUENCE) {}
```

Comment:
about frequency: I'm a bit sad that we need to add a parameter for all commands, but used only for command used in `usage`. Two options:
- inherit from Command class and add frequency in subclass
- don't allow user to configure frequency and compute it based on other parameters (timeout for instance)